### DISCUSSION TITLE
Auto-dim empty display(s) if have multiple displays to extend display lifetimes

### DISCUSSION THREAD

**Discussion by techdebtcreator** [MonitorControl/MonitorControl#1504] [Ideas]
For a multi-display setup, it would be nice to auto-dim unused displays to a defined brightness to increase their lifetime. I'm imagining this could be a general setting (turned off by default).  "Unused" could be defined as having no windows overlapping the display boundaries, so if user drags window into a display it will return to its normal brightness.

**Comment by techdebtcreator** [comment]
I have no Swift / MacOS programming experience, but my hunch is that it would involve at least this notification https://developer.apple.com/documentation/appkit/nswindow/1419645-didmovenotification w/ possibly a delay on how often the check is performed to not overly consume resources.

**Comment by waydabber** [comment]
This feature won't be added to MonitorControl probably as the app is no longer actively developed, it reached feature maturity (that was the team decision). You can however add this here and will tag it as an `enhancement` so it could be implemented in a future version.

https://github.com/waydabber/BetterDisplay/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
